moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -145,9 +144,11 @@ class Robot
    * @brief execute a given trajectory
    * @param traj_name - name of trajectory to be executed (must be filled with
    * prior calls to "addTrajPoint".
-   * @return - true if desired trajectory was executed.
+   * \throws <std::runtime_error> (Execution failure)
```

Comment:
I think it can be handled like this.

try{
// do something
}
catch(std::exception& ex)
{
std::cout << "Exception: " << ex.what() << "\n";
// divide into multiple cases how a particular exception(ex.what()) should be handled.
}